{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "python3",
                "purl": "pkg:rpm/vmware/python3?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "3.7.5-9.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2021-3.0-0192",
    "modified": "2023-03-31T02:19:57Z",
    "published": "2021-02-11T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-192"
        }
    ],
    "related": [
        "CVE-2021-3177"
    ]
}